scripts/getDependencies.pl

```diff
@@ -26,9 +26,11 @@
 my $path;
 # define task
 my $task = "default";
+my $dependencyList;
```

Comment:
You probably want to initialize it to "". Otherwise the code might break when there's no dependencyList option.

Reply:
Could we default to all? Only change if `dependencyList` is set by users. In this way, all the getDependency jobs do not need to update. i.e., https://ci.eclipse.org/openj9/job/test.getDependency/